todo-app/src/Components/todoItem.js

```diff
@@ -0,0 +1,21 @@
+//jshint esnext: true
+
+import React from 'react';
+import './styles/todoItem.css';
+
+
+export default class TodoItem extends React.Component {
+    render (){
+        const {description, deadline, done, id } = this.props.data;
+        return (
+            <li className="todoItem">
+                <div>
+                    <span>{deadline}</span>
+                    <label for={`item${id}`} className={done ? 'markDone' : 'markUndone'}></label>
+                    <input type="checkbox" id={`item${id}`} onClick={() => this.props.handleIsItDone(this.props.data)}/>
```

Comment:
It's a matter of personal preference, but if you're declaring `item${id}` more than once, you might declare it in your `render` function before the return, like:

    const inputID = `item${id}`